altair/utils/schemapi.py

```diff
@@ -816,36 +826,41 @@ def to_dict(self, validate=True, ignore=None, context=None):
 
     def to_json(
         self,
-        validate=True,
-        ignore=None,
-        context=None,
-        indent=2,
-        sort_keys=True,
+        validate: bool = True,
+        ignore: Optional[List[str]] = None,
+        context: Optional[Dict[str, Any]] = None,
+        indent: int = 2,
+        sort_keys: bool = True,
```

Comment:
While we're changing signatures, how would you feel about moving `indent` and `sort_keys` above `ignore` and `context`. Like this:

```python
    def to_json(
        validate: bool = True,
        indent: int = 2,
        sort_keys: bool = True,
        ignore: Optional[List[str]] = None,
        context: Optional[Dict[str, Any]] = None,
```

I doubt this would be breaking in practice as `ignore` and `context` aren't intended for use by end users, and even if they are being used it would be even more unusual to specify `validate` `ignore` and `context` as positional args.

This is top of mind because I'd rather add the format argument in #3071 before `ignore` and `context`.